.mergify.yml

```diff
@@ -0,0 +1,10 @@
+pull_request_rules:
+  - name: automatic merge for master when CI passes and 1 reviews and has label S:merge
+    conditions:
+      - "#approved-reviews-by>=1"
+      - "status-success=ci/circleci: test-cover"
+      - base=master
+      - label=S:merge
```

Comment:
Maybe `automerge`? Just to make it clear that it's not advisory, it actually causes things to happen.

Reply:
ACK `automerge`.